src/components/EolPage.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { format, isAfter, isSameDay, parseISO } from 'date-fns';
+import { ja, ko } from 'date-fns/locale';
+import { useStaticQuery, graphql } from 'gatsby';
+import { compareVersions } from 'compare-versions';
+import getAgentName from '../utils/getAgentName.js';
+import getEOLDate from '../utils/getEOLDate.js';
+
+const releaseNotesQuery = graphql`
+  query {
+    allMdx(
+      filter: {
+        fileAbsolutePath: {
+          regex: "/src/content/docs/release-notes/.*(?<!index).mdx/"
+        }
+      }
+    ) {
+      nodes {
+        frontmatter {
+          releaseDate
+          subject
+          version
+        }
+      }
+    }
+  }
+`;
+
+const EolPage = ({ agent, locale = 'en' }) => {
+  const { allMdx } = useStaticQuery(releaseNotesQuery);
+  const releaseNotesJson = allMdx.nodes.map((note) => {
+    return {
+      agent: getAgentName(note.frontmatter.subject),
+      date: note.frontmatter.releaseDate,
+      eolDate: getEOLDate(note.frontmatter.releaseDate),
```

Comment:
`sortDateDesc`, `formatDate`, and `isSupportedVersion` all call `parseISO` on the date that they receive. would it make sense to call `parseISO` here instead to avoid repeatedly calling it?